### DISCUSSION TITLE
Domain name change after I change my deploy branch

### DISCUSSION THREAD

**Discussion by jennyberry** [community/community#31482] [Other Feature Feedback, Questions, & Ideas] [answered]
After committing to gh-pages and I didn't see any change on my website, I thought I set the wrong branch to deploy so I change to another branch. 
However, the problem didn't solve so I change it back to the first branch. After that, I found my domain name (https://example.com/) has been changed to (https://project.github.io/). 
I'm not the owner of the code so I couldn't tell if they use any custom domain name or not.

But there any way to change the domain name back to (https://example.com/)

Thank you a lot for your help!!!!!!

**Comment by jfullstackdev** [comment]
I think you are on the free account, so the repo source of the GitHub Pages you are referring to is also public

if you can share it, then that will be much better

**Comment by airtower-luna** [comment, marked as answer]
> However, the problem didn't solve so I change it back to the first branch. After that, I found my domain name (https://example.com/) has been changed to (https://project.github.io/).

Was there a `CNAME` file on the old branch that's missing on the new one? AFAIK that's what defines a custom domain to use.
